pages/support.html.txt

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Support-page</title>
+</head>
+<body>
+
```

Comment:
Delete all empty lines and change file extension in file editor:D

Reply:
No, ty